Trumpf.Coparoo.Web/Controls/Table/TableBody.cs

```diff
@@ -0,0 +1,34 @@
+﻿namespace Trumpf.Coparoo.Web.Controls.Table
+{
+    using OpenQA.Selenium;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// Table body control object.
+    /// </summary>
+    public class TableBody : ControlObject
```

Comment:
I suggest to introduce an abstract base class `Table<RowType>` with `IEnumerable<RowType> Rows => FindAll<RowType>();` and `public RowType RowAt(int index)`. The three derived types then only override the `SearchPattern` property (nothing else in them).